test_load_script is language dependent

#### Problem Description
The `test_load_script` test uses language heuristics to check for a "nonexistent" file error. This makes the test fail on a non-English machine: https://github.com/mitmproxy/mitmproxy/blob/6156cbc0b15b4a35fcdaa931d6d906b3b3e8233f/test/mitmproxy/addons/test_script.py#L28

A better solution would be to check for the Python error `FileNotFoundError` instead

```python
_____________________________________________________________________________________ test_load_script ______________________________________________________________________________________

tmp_path = PosixPath('/tmp/pytest-of-daniel/pytest-82/test_load_script0'), tdata = <mitmproxy.utils.data.Data object at 0x77a985c72dd0>
caplog = <_pytest.logging.LogCaptureFixture object at 0x77a985c70490>

    def test_load_script(tmp_path, tdata, caplog):
        ns = script.load_script(
            tdata.path("mitmproxy/data/addonscripts/recorder/recorder.py")
        )
        assert ns.addons
    
        script.load_script("nonexistent")
>       assert "No such file or directory" in caplog.text
E       assert 'No such file or directory' in 'ERROR    mitmproxy.addons.script:script.py:69 error in script nonexistent\nTraceback (most recent call last):\n  File "/home/daniel/Sources/mitmproxy/mitmproxy/addons/script.py", line 38, in load_script\n    loader.exec_module(m)\n  File "<frozen importlib._bootstrap_external>", line 936, in exec_module\n  File "<frozen importlib._bootstrap_external>", line 1073, in get_code\n  File "<frozen importlib._bootstrap_external>", line 1130, in get_data\nFileNotFoundError: [Errno 2] No existe el fichero o el directorio: \'nonexistent\'\n'
E        +  where 'ERROR    mitmproxy.addons.script:script.py:69 error in script nonexistent\nTraceback (most recent call last):\n  File "/home/daniel/Sources/mitmproxy/mitmproxy/addons/script.py", line 38, in load_script\n    loader.exec_module(m)\n  File "<frozen importlib._bootstrap_external>", line 936, in exec_module\n  File "<frozen importlib._bootstrap_external>", line 1073, in get_code\n  File "<frozen importlib._bootstrap_external>", line 1130, in get_data\nFileNotFoundError: [Errno 2] No existe el fichero o el directorio: \'nonexistent\'\n' = <_pytest.logging.LogCaptureFixture object at 0x77a985c70490>.text

test/mitmproxy/addons/test_script.py:28: AssertionError
------------------------------------------------------------------------------------- Captured log call -------------------------------------------------------------------------------------
ERROR    mitmproxy.addons.script:script.py:69 error in script nonexistent
Traceback (most recent call last):
  File "/home/daniel/Sources/mitmproxy/mitmproxy/addons/script.py", line 38, in load_script
    loader.exec_module(m)
  File "<frozen importlib._bootstrap_external>", line 936, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1073, in get_code
  File "<frozen importlib._bootstrap_external>", line 1130, in get_data
FileNotFoundError: [Errno 2] No existe el fichero o el directorio: 'nonexistent'

```

#### Steps to reproduce the behavior:
1.  Change your system language to another that isn't English
2. Run mitmproxy tests

#### System Information
Mitmproxy: 11.0.0.dev
Python:    3.11.8
OpenSSL:   OpenSSL 3.2.1 30 Jan 2024
Platform:  Linux-6.7.8-zen1-1-zen-x86_64-with-glibc2.39

--- a/test/mitmproxy/addons/test_script.py
+++ b/test/mitmproxy/addons/test_script.py
@@ -25,7 +25,7 @@ def test_load_script(tmp_path, tdata, caplog):
     assert ns.addons
 
     script.load_script("nonexistent")
-    assert "No such file or directory" in caplog.text
+    assert "FileNotFoundError" in caplog.text
 
     (tmp_path / "error.py").write_text("this is invalid syntax")
     script.load_script(str(tmp_path / "error.py"))

